lib/bulkPrefixer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Ah.  OK.  Makes sense.

Also, could you jump on the group chat.  There's much loader chat going on about how to load the TIGER data.